packages/typespec-ts/src/modular/emitModels.ts

```diff
@@ -128,6 +130,9 @@ function emitType(context: SdkContext, type: SdkType, sourceFile: SourceFile) {
     if (isAzureCoreErrorType(context.program, type.__raw)) {
       return;
     }
+    if (isOrExtendsHttpFile(context.program, type.__raw!)) {
```

Comment:
I wonder if we could use usage to filter the HttpPart models out?

Reply:
We can't because tcgc doesn't set UsageFlags.MultipartFormData for `Http.File`. It only does that for the actual multipart body model.